src/main/resources/hibernate.cfg.xml

```diff
@@ -12,7 +12,10 @@
         <property name="hibernate.connection.password">Vremere_a</property>
         <property name="show_sql">true</property>
         <property name="hibernate.hbm2ddl.auto">create-drop</property>
+        <property name="hibernate.enable_lazy_load_no_trans">true</property>
```

Comment:
https://vladmihalcea.com/the-hibernate-enable_lazy_load_no_trans-anti-pattern/

Reply:
del